x-pack/plugins/apm/public/components/app/ServiceDetails/MetricsChart.tsx

```diff
@@ -52,9 +52,10 @@ function getYTickFormatter(chart: GenericMetricsChart) {
   switch (chart.yUnit) {
     case 'bytes': {
       const max = Math.max(
-        ...chart.series.flatMap(series =>
-          series.data.map(coord => coord.y || 0)
-        )
+        // TODO: Use flatMap
+        ...chart.series
+          .map(series => series.data.map(coord => coord.y || 0))
+          .reduce((prev, current) => prev.concat(current))
       );
```

Comment:
This will work just fine for now, but it would be nice to avoid the extra iteration while also splitting the args to Math.max into smaller chunks to mitigate the risk of the maximum function args limit (depending on js engine):
```
const max = Math.max(
  ...chart.series.map(({ data }) => Math.max(...data.map(({ y }) => y)))
);
```

Reply:
Thanks for the comment, I'll make this change